src/components/file/examples/file-custom.tsx

```diff
@@ -0,0 +1,51 @@
+import { FileInfo } from '@limetech/lime-elements';
```

Comment:
We usually leave off the `limel-` part in the commit message scope. So for consistency, I suggest renaming the commit `feat(file): support custom icon and icon-color` 🙂

Reply:
Fixed :)